src/app/hero-search/hero-search.component.css

```diff
@@ -0,0 +1,47 @@
+    /* HeroSearch private styles */
```

Comment:
Think we could clean up the magic numbers, BEM this, maybe split it up

Reply:
When Gav says "magic numbers" he means numbers like `1.2rem` or `.5rem`

Create too many instances of these and you'll end up with a lack of consistent sizes/spacing across your app, or be using more effort than necessary to update all the various files in an attempt to keep it consistent, one approach we use to avoid this is using a centralised set of  [variables](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties). These are sometimes referred to as "design tokens".

For example `1.2rem` seems to be consistently used so it could be pulled out into a `var(--font-size-lg)` (`--font-size` would be `1rem`).